servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/DefaultHttpLoadBalancerFactory.java

```diff
@@ -171,7 +309,41 @@ public <T> Publisher<? extends T> transportEventStream(final HttpEventKey<T> eve
 
         @Override
         public Single<StreamingHttpResponse> request(final StreamingHttpRequest request) {
-            return delegate.request(request);
+            return Single.defer(() -> {
+                if (tracker == null) {
+                    return delegate.request(request).shareContextOnSubscribe();
+                }
+
+                final RequestTracker theTracker = new AtMostOnceDeliveryRequestTracker(tracker);
+                final long startTime = theTracker.beforeStart();
+
+                return delegate.request(request)
+                        .map(response -> {
```

Comment:
`map` should be applied after `BeforeFinallyHttpOperator`, see `AbstractLifecycleObserverHttpFilter` and borrow a comment from there.